pages/index.vue

```diff
@@ -158,8 +158,8 @@
             </div>
             <OverlayModal :show="deleteDraftPopUpShow" :full="false" @close="deleteDraftPopUpShow = false">
                 <DecisionPopUp
-                    imgSrc="http://localhost:3000/catalog/Characters/single/Barista%20pouring4.png?raw=true"
-                    title="Poof, Your hard work disappears"
+                    imgSrc="./Barista_pouring4.png"
```

Comment:
For me it is uncommon to load the images from the public folder over `./`. I am used to just `/`. Since it works we can leave it like that, just wanted to note it.